install.sh

```diff
@@ -27,6 +27,6 @@
 
 # package tar.gz/zip
 gradle clean -Pdev=true -Pjdk=1.8 dist tar zip
-
+gradlew installPlugin
 # package jar
-gradle clean -Pdev=true -Pjdk=1.8 jar
\ No newline at end of file
+#gradle clean -Pdev=true -Pjdk=1.8 jar
```

Comment:
This script seems to be getting old, you can just delete this script. If you want to build the project, please refer to [this documentation](https://eventmesh.apache.org/docs/instruction/runtime/#2-build-binary-distribution).